frontend/src/components/LeftPanel/NavBar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { FaUserAlt } from "react-icons/fa";
+function NavBar() {
+  return (
+    <div className='border-2 border-black w-20 flex flex-col items-center'>
+      {/* TODO: Need to prepare a logo and favicon */}
+      <div className='w-full h-16 border-2 border-orange-300'>
+        Logo
+      </div>
+      <div className='h-4/6 my-auto flex flex-col justify-evenly items-center'>
+
+        {/* TODO:  Need to provide icons for following
+        Basic details, address, social media, work ex, education, awards, publications, certifications, skills, languages, interests, projects, references */}
+
+        <FaUserAlt />
```

Comment:
Will there be links that move the view to that location as well? If so, then let's just put all the icons and links in an array of objects and map that here, hard-coded values make it inflexible to add more sections in the future.

The object can look like:

```js
{
    "href": "some-link",
    "icon": "icon component",
}
```